.github/workflows/publish_pypi.yml

```diff
@@ -30,30 +35,52 @@ jobs:
         path: dist/*.tar.gz
 
   publish_to_TestPyPI:
-    needs: [make_sdist]
-
-    # The 'success' check for triggering workflow may be redundant since we depend on make_sdist, but keep it just in case
-    #if: github.event_name == 'release' && github.event.action == 'published' && ${{ github.event.workflow_run.conclusion == 'success' }}
-    # TEMP for testing
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
-
+    needs: make_sdist
     name: Publish to TestPyPI
```

Comment:
Do you want to change the name of this?

Reply:
I guess the name should also be different to those of the jobs below (`Publish to PyPI` and `Publish to TestPyPI`). How about `Publish to PyPI or TestPyPI`?

Reply:
Whoops yeah changed now